### ISSUE TITLE
AC temp only updates when device power is active

### ISSUE THREAD

**Issue by 0Styless** [georgezhao2010/midea_ac_lan#335]
### HA core version

2023.9.2

### Intergration version

v0.3.21

### Device type and model

Air Conditioner 00000Q1A / Air Conditioner 00000Q13

### Used App

Carrier Air Conditioner

### The description of problem

Hey there,

this is more a question than a bug report - because even with the official "Carrier Air Conditioner" android app, this doesn't seem to work.

TLDR: Temperature sensors of the AC devices only update their values if device power switches to active/on. Is there any endpoint which I can trigger with Home-Assistant, curl, wget, you name it - to force an update of those values without turning the device on and off again?

**Comment by georgezhao2010** [comment]
> ### The description of problem
> Hey there,
> 
> this is more a question than a bug report - because even with the official "Carrier Air Conditioner" android app, this doesn't seem to work.
> 
> TLDR: Temperature sensors of the AC devices only update their values if device power switches to active/on. Is there any endpoint which I can trigger with Home-Assistant, curl, wget, you name it - to force an update of those values without turning the device on and off again?


In fact, these values can also be updated when the device power is off, but they will display as invalid values like ‘FF’, which I have filtered out. 

Moreover, different devices are not the same in terms of temperature updates, some devices always provide valid values, while others only provide valid values when the power is on. Unfortunately, your device is like mine, it can only provide valid values when the power is on.

**Comment by 0Styless** [comment]
Sad - but thank you to take your time to answer this question!
I will close this one.
